anchore_engine/analyzers/modules/32_java_packages.py

```diff
@@ -27,27 +17,44 @@
 try:
     config = anchore_engine.analyzers.utils.init_analyzer_cmdline(sys.argv, analyzer_name)
 except Exception as err:
-    print str(err)
+    print(str(err))
     sys.exit(1)
 
 imgname = config['imgid']
 imgid = config['imgid_full']
 outputdir = config['dirs']['outputdir']
 unpackdir = config['dirs']['unpackdir']
 
-def process_java_archive(prefix, filename, inZFH):
+
+def parse_properties(file):
```

Comment:
I didn't know about the library. If the maintainers are ok with using it, it'd also be good to bring in a proper manifest file parser since the existing one doesn't account for long attribute values (which get line wrapped).

Reply:
@jvz - We're happy to merge this in, and then perhaps investigate the use of pyjavaproperties for future inclusion if it looks like it could fit the requirement (- what would be your preference @jvz (merge as is, work on additional / alternate implementations moving forward, or work on alternate implementation first?)

In a nutshell, we'd be okay bringing in this dependency if the author of the PR would like to write an implementation based on it and finds the dependency of high quality.

Reply:
@jvz @MarkEWaite - we were looking into the pyjavaproperties a bit further, and one thing that is a problem is that it appears to have some open PRs from 2014/2015 that still have not been merged, and one of them is 'support for python 3'.  We're going to be moving to python 3 very soon, and so we would like to ensure that any dep we bring in is py3 compatible moving forward.

Reply:
You can merge this as is. Improvements to properties files and manifest file parsing can be added separately, though the existing logic should be adequate for the properties being checked.

Reply:
Merge as-is seems very reasonable to me.  The question about pyjavaproperties was not intended to be a blocker and if it doesn't support Python 3, that seems like a very good reason to not use pyjavaproperties

Reply:
merged - thank you very much for the contribution and all for digging in!